src/transformers/models/auto/configuration_auto.py

```diff
@@ -174,6 +174,7 @@
         ("nllb-moe", "NllbMoeConfig"),
         ("nougat", "VisionEncoderDecoderConfig"),
         ("nystromformer", "NystromformerConfig"),
+        ("olmo", "OLMoConfig"),
```

Comment:
```suggestion
        ("olmo", "OlmoConfig"),
```
we need camel casing on all classes

Reply:
[c7f4e74](https://github.com/huggingface/transformers/pull/29890/commits/c7f4e7489836d43b1d6a34cf47eb1de1120d1961)